.github/workflows/marine-rs-sdk-test.yml

```diff
@@ -0,0 +1,53 @@
+name: "marine-rs-sdk-test"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  service:
+    name: "Build and test Rust service"
+    runs-on: builder
+
+    env:
+      RUST_BACKTRACE: 1
+      CARGO_TERM_COLOR: always
+
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2022-06-30
+          target: wasm32-wasi
```

Comment:
This project builds to host target. It does not need `wasm32-wasi` target available.